client/signup/steps/plans/index.jsx

```diff
@@ -255,6 +266,10 @@ export class PlansStep extends Component {
 			}
 		}
 
+		if ( ! this.state.experimentLoaded ) {
+			return null;
```

Comment:
It would be better if we can show a loading spinner here. However, it feels like the promise would resolve immediately so I've never seen the blank that should be here.

Reply:
Since the experiment is pre-fetched, it would be unlikely for the experiment to not be loaded by the time the user reaches this step.. So the rendering should be immediate.